components/Common/SubscribeForm/index.jsx

```diff
@@ -101,41 +100,33 @@ const SubscribeForm = ({isCompactView}) => {
     setAfterFormSubmitMessage('');
     setFormSuccess(null);
 
-    let host = 'https://dev.us5.list-manage.com/subscribe/post-json?';
-    let data = {
-      u: process.env.NEXT_PUBLIC_MAILCHIMP_SUBSCRIBE_FORM_U,
-      id: process.env.NEXT_PUBLIC_MAILCHIMP_ID,
-      c: process.env.NEXT_PUBLIC_MAILCHIMP_SUBSCRIBE_FORM_C,
+    const host = '/api/subscribe';
```

Comment:
Should we use the env variable?

Reply:
Submission to hubspot happens on server so this isn't necessary here.